docs/content/getting-started/faq.md

```diff
@@ -0,0 +1,114 @@
+# FAQ
+
+## Why is Traefik Answering XXX HTTP Response Status Code ?
+
+Traefik is a dynamic reverse proxy,
+and while the documentation often demonstrates configuration options through file examples,
+the core feature of Traefik is its ability to be provided with configuration dynamically.
+
+The file provider allows providing [dynamic configuration](../configuration-overview/#the-dynamic-configuration) through a file or directory.
+This is different from the [static configuration](../configuration-overview/#the-static-configuration),
+which can also be provided by a file.
+
+For HTTP configurations, the EntryPoint concept matches the Transport Layer (TCP),
+while the Router concept matches the Presentation (TLS) and Application layers (HTTP).
+
+An EntryPoint is not tied to the protocol HTTP or HTTPS,
+as one can attach a TLS router or a non-TLS router to the same EntryPoint.
+
+***Therefore, in this dynamic context, when an EntryPoint is configured, 
+Traefik is handling traffic on the EP port without the guarantee to match any configured route.***
```

Comment:
```suggestion
***Therefore, in this dynamic context, the static configuration of an EntryPoint does not give any hint whatsoever about how the traffic going through that EntryPoint is going to be routed. Or whether it's even going to be routed at all, i.e. whether there is a Router matching the kind of traffic (a request, or otherwise) going through it. ***
```